app/views/shared/_cohort_overview.html.erb

```diff
@@ -0,0 +1,80 @@
+        <%= link_to t('.students_link'), send(students_path, *args), class: "p-2 hover:text-primary-500 hover:bg-primary-100" %>
+      </div>
+    </div>
+  </section>
+  <%
+    total_students_count = @presenter.counts[:total]
+    milestone_completion_status = @presenter.milestone_completion_status if total_students_count > 0
+  %>
+  <section class="bg-white">
+    <div class="max-w-5xl mx-auto p-4 md:p-6">
+      <div class="mt-4 grid grid-cols-1 md:grid-cols-3 gap-5">
+        <div class="p-5 bg-white flex flex-col gap-2 border border-gray-300 rounded-lg">
+          <p class="text-sm text-gray-600"><%= t('.total_students') %></p>
+          <p class="text-3xl font-bold"><%= total_students_count %></p>
+          <%= link_to send(students_path, *args), class: "text-sm font-medium text-primary-500 hover:text-primary-800 hover:underline" do %>
+            <span><%= t('.view_all')%></span><i class="if i-arrow-right-regular rtl:rotate-180 inline-block text-primary-500 ms-2" aria-hidden="true"></i>
+          <% end %>
+        </div>
+        <div class="p-5 bg-white flex flex-col gap-2 border border-gray-300 rounded-lg">
+          <p class="text-sm text-gray-600"><%= t('.students_completed') %></p>
+          <p class="text-3xl font-bold"><%= @presenter.counts[:completed] %></p>
+          <%= link_to send(students_path, *args, course: "Completed"), class: "text-sm font-medium text-primary-500 hover:text-primary-800 hover:underline" do %>
+            <span><%= t('.view_all')%></span><i class="if i-arrow-right-regular rtl:rotate-180 inline-block text-primary-500 ms-2" aria-hidden="true"></i>
+          <% end %>
+        </div>
+      </div>
+      <% if (total_students_count > 0 or milestone_completion_status.present?) && (@presenter.milestone_targets.present?) %>
+        <div class="p-4 bg-gray-50 rounded-lg mt-8">
+          <div class="max-w-5xl mx-auto">
+            <h3 class="text-xs font-semibold"><%= t('.student_distribution_milestone') %></h3>
+            <div class="mt-4">
+              <div class="grid gap-x-8 gap-y-6">
+                <% @presenter.milestone_targets.each do |target| %>
+                  <% milestone_target_stats = milestone_completion_status[target.id] %>
+                  <div>
+                    <div class="flex justify-between">
+                      <p class="text-xs font-medium"><%= "M#{target.milestone_number}: #{target.title}" %></p>
+                      <div class="flex space-x-2">
+                        <p class="text-xs font-medium text-gray-500"><%= "#{milestone_target_stats[:students_count]}/#{total_students_count}" %></p>
+                        <p class="text-center text-xs font-medium rounded-sm px-1 bg-gray-200"><%= "#{milestone_target_stats[:percentage]}%" %></p>
+                      </div>
+                    </div>
+                    <%= link_to send(students_path, milestone_completed: "#{target.id};M#{target.milestone_number}: #{target.title}") do %>
```

Comment:
If we pass the generated path from the calling site, we can avoid the use of `send`

Reply:
Okay got it. Let's send the path as symbol, then from the calling site and generate the path here using the current setup.